docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,293 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+****************************************
+Get a List of Course Blocks in a Course
+****************************************
+
+The endpoint to get a list of course blocks in a course is
+``/api/courses/v1/blocks/`` with required ``course_id`` parameter.
+
+=====================
+Use Case
+=====================
+
+Get a list of course blocks in a specified course. Response results depend on
+the viewing user's permissions level within a course, as well as group
+membership and individual allowances (such as due date extensions), if any.
+
+=====================
+Request Format
+=====================
+
+GET /api/courses/v1/blocks/?course_id=<course_id>
+
+Example:
+
+GET /api/courses/v1/blocks/?course_id=edX%2FDemoX%2FDemo_Course
+&all_blocks=true&requested_fields=graded,format,student_view_multi_device
+
+
+.. _Blocks Query Parameters:
+
+=====================
+Query Parameters
+=====================
+
+* all_blocks: (boolean) Provide a value of ``true`` to return all blocks.
+  Returns all blocks only if the requesting user has course staff permissions.
+  Blocks that are visible only to specific learners (for example, based on
+  group or cohort membership, or randomized content from content libraries)
+  are not included. If ``all_blocks`` is not specified, you must specify the
```

Comment:
Correction: Blocks that are visible only to specific learners are _all_ included when "all_blocks" is specified.